docs/operations-guide/running-the-metabase-jar-file.md

```diff
@@ -1,49 +1,50 @@
-# Running the Metabase Jar File
+# Running the Metabase JAR-file
 
-To run the Metabase jar file you need to have Java installed on your system. Currently Metabase requires Java 8 or higher and will work on either the OpenJDK or Oracle JRE.
+To run the Metabase JAR-file you need to have Java Runtime Environment (JRE) installed on your system.
 
 ### Download Metabase
 
-If you haven't done so already the first thing you need to do is [Download Metabase](https://metabase.com/start/jar.html).  Simply save the .jar file to a folder on your system where you wish to run Metabase.
-
+If you haven't done so already the first thing you need to do is [Download Metabase](https://metabase.com/start/jar.html). Simply save the JAR-file to a folder on your system where you wish to run Metabase.
 
 ### Verify Java is installed
```

Comment:
Can we point toward the java versions page for this info instead, so the info is just in one place?